lib/monetary.ts

```diff
@@ -0,0 +1,174 @@
+import Big from 'big.js';
+
+
+type BigOrNum = Big | number;
```

Comment:
Could we support `BigNumberish` from `ethers`?

Reply:
Changed to `BigSource` from `big.js`  
I guess changing to BigNumberish will depend on whether we switch to ethers or not.